helm/templates/validator-job.yaml

```diff
@@ -28,10 +28,23 @@ spec:
           env:
             {{- include "cloudzero-agent.validatorEnv" . | nindent 12 }}
           command:
-            - /app/cloudzero-agent-validator
```

Comment:
it might be worth renaming this job and the related named templates, etc, from `validator` to `cluster-config` or similar